app/routers/profile.py

```diff
@@ -108,6 +128,28 @@ async def update_profile(
     return RedirectResponse(url=url, status_code=HTTP_302_FOUND)
 
 
+async def get_current_user():
+    user = get_placeholder_user()
```

Comment:
Consider `return get_placeholder_user()`.
I think that a documention and typing could help here.